recipe/meta.yaml

```diff
@@ -21,36 +21,44 @@ requirements:
   host:
     - git
     - python >=3.7
-    - pip
+    - pip !=21.3
     - pytest-runner
     - setuptools_scm
   run:
     # esmvaltool
     - python >=3.7
     - iris >=3.1.0
     - python-stratify
-    - scipy
+    - scipy >=1.6
     # Normally installed via pip:
+    - cartopy
     - cftime
     - cf-units >=3.0.0
+    - compilers
```

Comment:
What is the `compilers` dependency needed for? This looks odd.

Reply:
Fine for me, I just copied it from the [`environment.yml`](https://github.com/ESMValGroup/ESMValCore/blob/ae1f43fcf70ad89efbd2cc31ed37d862e49c82a1/environment.yml#L9) of  ESMValCore.

Reply:
My trust in the `environment.yml` file is limited. I think the main reason one might want to have `compilers` is to allow for installation of packages from pypi that need compiling some bits. But since our stated goal is to avoid that and have everything come from conda-forge, and because we have made some good progress on that, I suggest we remove compilers here and see if things still work out. If they don't, we can see how to address that, be it by adding packages to conda-forge or by re-adding compilers.